Api/Program.cs

```diff
@@ -1,17 +1,29 @@
+using Npgsql;
 using System.Data;
-
 using Api.Application;
-
+using Api.Domain.Services;
+using Api.Domain;
 using FluentValidation.AspNetCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
+var connectionString = builder.Configuration.GetConnectionString("DefaultConnection") ??
+                       "Host=localhost;Database=expense_tracker_group_5_postgres;Username=user;Password=password";
+
+builder.Services.AddTransient<IDbConnection>(sp => 
```

Comment:
remove the code to initialize the Connection with the DB, you don't need it anymore

Reply:
I already implemented that in this line of code moving responsibilities to another folder:
![Screenshot from 2024-09-27 10-45-45](https://github.com/user-attachments/assets/45c89478-3cbe-44c9-b0f8-506e945c3f4f)